app/components/keyword/keywordDirective.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+    angular
+    .module('search_keyword_component', [])
+    .directive('keyword', function keyword() {
+        return {
+            scope: {
+                tag: '@',
+                remove: '=',
```

Comment:
why is there a boolean for if the tag is removed or not? could you just remove it from the scope?